contracts/ROSCA.sol

```diff
@@ -600,15 +606,18 @@ contract ROSCA {
    * this sends all the funds to the foreperson by selfdestructing this contract.
    */
   function emergencyWithdrawal() onlyFromForeperson onlyIfEscapeHatchActive external {
-    LogEmergencyWithdrawalPerformed(getBalance(), currentRound);
+	  LogEmergencyWithdrawalPerformed(getBalance(), currentRound);
+    bool failedSendingFunds = false;
```

Comment:
please use different var name. It's misleading because `failedSendingFunds = tokenContract.transfer(foreperson, balance);` means the opposite

Reply:
done